pgjdbc/src/main/java/software/aws/rds/jdbc/postgresql/auth/AwsIamAuthenticationPlugin.java

```diff
@@ -156,7 +155,7 @@ private Region getRdsRegion(final String hostname) throws PSQLException {
     return regionOptional.get();
   }
 
-  private static class TokenInfo {
+  public static class TokenInfo {
```

Comment:
do you need this to be public? Or just some public accessors?

Reply:
Will make package private.